### PR TITLE
Document filter dependencies

### PR DESCRIPTION
**Pull request by LukeTowers**
Refs: https://github.com/octobercms/october/pull/4764

### CODE REVIEW

File: backend-lists.md [hunk 1]
```diff
@@ -452,6 +452,45 @@ Option | Description
 **nameFrom** | if filtering by multiple items, the attribute to display for the name, taken from all records of the `modelClass` model.
 **default** | can either be integer(switch,checkbox,number) or array(group,date range,number range) or string(date).
 **permissions** | the [permissions](users#users-and-permissions) that the current backend user must have in order for the filter scope to be used. Supports either a string for a single permission or an array of permissions of which only one is needed to grant access.
+**dependsOn** | a string or an array of other scope names this scope [depends on](#filter-scope-dependencies), when the other scopes are modified, this scope will update.
```
Comment (bennothommo) [inline backend-lists.md]: @LukeTowers Grammar fix:
```suggestion
**dependsOn** | a string or an array of other scope names that this scope [depends on](#filter-scope-dependencies). When the other scopes are modified, this scope will update.
```

File: backend-lists.md [hunk 2]
```diff
@@ -452,6 +452,45 @@ Option | Description
...
+
+<a name="filter-scope-dependencies">
+### Filter Dependencies
+
+Filter scopes can declare dependencies on other scopes by defining the `dependsOn` [scope option](#filter-scope-options) which provides a more robust server side solution for updating scopes when their dependencies are modified. When the scopes that are declared as dependencies change, the defining scope will update using the AJAX framework. This provides an opportunity to  change the available options to be provided to the scope. Examples below:
```
Comment (bennothommo) [inline backend-lists.md]: @LukeTowers Grammar fix:
```suggestion
Filter scopes can declare dependencies on other scopes by defining the `dependsOn` [scope option](#filter-scope-options), which provides a more robust server-side solution for updating scopes when their dependencies are modified. When the scopes that are declared as dependencies change, the defining scope will update using the AJAX framework. This provides an opportunity to change the available options to be provided to the scope. Examples below:
```

File: backend-lists.md [hunk 3]
```diff
@@ -452,6 +452,45 @@ Option | Description
...
+        
+    city:
+        label: City
+        type: group
+        conditions: city_id in (:filtered)
+        modelClass: October\Test\Models\Location
+        options: getCityOptions
+        dependsOn: country
+
+In the above example the `city` scope will refresh when the `country` scope has changed. Any scope that defines the `dependsOn` property will be passed all current scope objects for the Filter widget (which includes their current values) as an array keyed by the scope names:
```
Comment (bennothommo) [inline backend-lists.md]: @LukeTowers Grammar fix:
```suggestion
In the above example, the `city` scope will refresh when the `country` scope has changed. Any scope that defines the `dependsOn` property will be passed all current scope objects for the Filter widget (which includes their current values) as an array that is keyed by the scope names:
```

**Comment by LukeTowers** [comment]
Thanks @bennothommo, copied it pretty much verbatim from the field dependencies section.

**Pull request merged**